packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -58,15 +73,31 @@ const SettlementHandlerFacetI = M.interface('SettlementHandler', {
 export const prepareResolverKit = (
   resolverZone: Zone,
   zcf: ZCF,
-  vowTools: VowTools,
+  {
+    vowTools,
+    storageNode: portfoliosNode,
+    marshaller,
+  }: {
+    vowTools: VowTools;
+    storageNode: Remote<StorageNode>;
+    marshaller: Marshaller;
+  },
 ) => {
+      const pendingTxsNode = E(portfoliosNode).makeChildNode('pendingTxs');
```

Comment:
`'pendingTxs'` is part of the external interface; I suggest declaring a constant in `types.ts`

Reply:
and as discussed, have the caller make this child. POLA